src/identity/storage/getKeyValueInterfaceFromResourceStore.ts

```diff
@@ -0,0 +1,88 @@
+import { BasicRepresentation } from '../../ldp/representation/BasicRepresentation';
+import type { Representation } from '../../ldp/representation/Representation';
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+import type { ResourceStore } from '../../storage/ResourceStore';
+import { APPLICATION_OCTET_STREAM } from '../../util/ContentTypes';
+import { readableToString } from '../../util/StreamUtil';
+
+export interface KeyValueInterface {
+  get: (resourceIdentifier: ResourceIdentifier) => Promise<unknown | undefined>;
+  set: (
+    resourceIdentifier: ResourceIdentifier,
+    payload: unknown,
+    options?: { expires?: Date },
+  ) => Promise<void>;
+  remove: (identifier: ResourceIdentifier) => Promise<void>;
+}
+
+export interface KeyValueInterfaceStorage {
+  payload: unknown;
+  expires?: string;
+}
+
+export function getKeyValueInterfaceFromResourceStore(
```

Comment:
This needs to be a class `KeyValueResourceStore` and this is its constructor.

CC @joachimvh for the other PR where we needed this

Reply:
Done, but I'd argue against the name.

Calling it `KeyValueResourceStore` would imply that it is a `ResourceStore`. It is not. It only passes data through to one as an interface. So, I made `PassthroughKeyValueStore`.